usr/share/rear/restore/TSM/default/400_restore_with_tsm.sh

```diff
@@ -10,8 +10,9 @@ for num in $TSM_RESTORE_FILESPACE_NUMS ; do
     test "${filespace:0-1}" == "/" || filespace="$filespace/"
     LogUserOutput "Restoring TSM filespace $filespace"
     Log "Running 'LC_ALL=$LANG_RECOVER dsmc restore $filespace $TARGET_FS_ROOT/$filespace -subdir=yes -replace=all -tapeprompt=no ${TSM_DSMC_RESTORE_OPTIONS[@]}'"
+    Log "File restore progression can be followed with tail -f \"$TMP_DIR/TSM_restore.log\""
     # Regarding usage of '0<&6 1>&7 2>&8' see "What to do with stdin, stdout, and stderr" in https://github.com/rear/rear/wiki/Coding-Style
-    LC_ALL=$LANG_RECOVER dsmc restore "$filespace" "$TARGET_FS_ROOT/$filespace" -subdir=yes -replace=all -tapeprompt=no "${TSM_DSMC_RESTORE_OPTIONS[@]}" 0<&6 1>&7 2>&8
+    LC_ALL=$LANG_RECOVER dsmc restore "$filespace" "$TARGET_FS_ROOT/$filespace" -subdir=yes -replace=all -tapeprompt=no "${TSM_DSMC_RESTORE_OPTIONS[@]}" 0<&6 1>"$TMP_DIR/TSM_restore.log" 2>&8
```

Comment:
In usr/share/rear/restore/NETFS/default/400_restore_backup.sh we have
<pre>
    (   case "$BACKUP_PROG" in
            (tar)
        ...
        esac >"${TMP_DIR}/${BACKUP_PROG_ARCHIVE}-restore.log"
</pre>
which results a `/tmp/rear.XXX/tmp/backup-restore.log` file
so that I suggest to use same kind of file name also for TSM i.e.
<pre>
... 1>"$TMP_DIR/TSM-restore.log" ...
</pre>
or perhaps even instead of the hardcoded `TSM`
some kind of BACKUP_PROG_ARCHIVE equivalent for TSM
if such a variable exists.